adr/database.md

```diff
@@ -0,0 +1,25 @@
+# Database Choice - MS-SQL
+
+## Status
+
+> Accepted
+
+> January 1, 2020
+
+## Context
+
+PIMS requires a database to store all property information.
+The data is relational, requiring contraints and must run within a linux docker container on OpenShift.
+Additionally it must be supported by Entity Framework Core 3.1.
+
+## Decision
+
+Originally the database generated for the SWU was with PostgreSQL, after further consideration it made more sense to tightly couple both MS-SQL with .NET Core.
+This will give us better performance and tighter integration with Entity Framework Core.
+It was decided to create a linux docker container to host the MS-SQL 2019 database.
+
+## Consequences
+
+There are no known negative consequences related to the PIMS project by making this decision.
```

Comment:
technically it makes our life harder when it comes to openshift backups - no existing lab project we can use.

Reply:
I've updated with a comment to reflect this.